engine/schema/src/main/resources/META-INF/db/schema-41120to41200.sql

```diff
@@ -34,4 +34,7 @@ INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`,
 INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`, `sort_order`) values (UUID(), 3, 'moveNetworkAclItem', 'ALLOW', 302) ON DUPLICATE KEY UPDATE rule=rule;
 INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`, `sort_order`) values (UUID(), 4, 'moveNetworkAclItem', 'ALLOW', 260) ON DUPLICATE KEY UPDATE rule=rule;
 
-UPDATE `cloud`.`async_job` SET `removed` = now() WHERE `removed` IS NULL;
\ No newline at end of file
+UPDATE `cloud`.`async_job` SET `removed` = now() WHERE `removed` IS NULL;
+
+-- New column for listManagementServers API call
+ALTER TABLE `mshost` ADD COLUMN `uuid` varchar(40) AFTER `name`;
```

Comment:
shouldn't you add a default value with the UUID() function?

Reply:
Sadly it is not possible to use `UUID()` as a default value (https://stackoverflow.com/questions/270309/can-i-use-a-function-for-a-default-value-in-mysql).

Reply:
Ok, but here I still have a question. I did not see any code to populate this `uuid` column in `mshost` table when you start to use this feature. I mean, after we upgrade the ACS to 4.12, how will we populate this `uuid` field for MShosts that already exist? Also,

Reply:
The code is there: https://github.com/apache/cloudstack/pull/2578/files#diff-8f89b2fb47915cb094656019c5010709R64 - I agree, it's small, it's a one liner.